mavros/launch/apm_config.yaml

```diff
@@ -193,6 +193,8 @@ vibration:
 # None
 
 odometry:
+  frame_tf:
+    desired_frame: "ned"
```

Comment:
why not just make this local_origin_ned

Reply:
it is. The idea is in the future handle other frame rotations (with offset for example). This is in a more general fashion